lib/sqlalchemy/dialects/oracle/cx_oracle.py

```diff
@@ -9,7 +9,7 @@
     :name: cx-Oracle
     :dbapi: cx_oracle
     :connectstring: oracle+cx_oracle://user:pass@host:port/dbname[?key=value&key=value...]
-    :url: http://cx-oracle.sourceforge.net/
+    :url: https://oracle.github.io/python-cx_Oracle/
```

Comment:
the merge conflicts were in the cx oracle dialect - but it doesn't show diffs because I decided that our changes were the most up-to-date (they were cherry-picked from a future release)